docs/design/metrics.md

```diff
@@ -97,10 +95,6 @@ http_request_duration_highr_seconds_count 201.0
 http_request_duration_seconds_count{handler="/v1/completions",method="POST"} 201.0
 ```
 
-### Multi-process Mode
-
-In v0, metrics are collected in the engine core process and we use multiprocess mode to make them available in the API server process. See <gh-pr:7279>.
```

Comment:
Metrics are still collected in the API server process, but multiprocess mode was reinstated by #17546 in order to share metrics state between API server processes